packages/wrangler/src/__tests__/pages/routes-transformation.test.ts

```diff
@@ -1,15 +1,15 @@
-import { toUrlPath } from "../../paths";
 import {
 	MAX_FUNCTIONS_ROUTES_RULES,
 	ROUTES_SPEC_DESCRIPTION,
 	ROUTES_SPEC_VERSION,
-} from "../constants";
```

Comment:
how did this ever work?

Reply:
oh wait...I see now. the tests files were moved. ok ok

Reply:
Yeah I just moved them into a directory where they would actually get run!